FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Tasks/TaskExecutor.cs

```diff
@@ -145,5 +116,55 @@ orderby dayGroups.Key
 
             return TaskResult.CreateFromTaskContext(taskContext);
         }
+
+        private async Task ExecuteInternalAsync(
+            JsonBatchData inputData,
+            TaskContext taskContext,
+            DateTime dateTime,
+            string continuationToken,
+            CancellationToken cancellationToken = default)
+        {
+            var parquetStreamResults = await _parquetDataProcessor.ProcessAsync(inputData, taskContext, cancellationToken);
```

Comment:
I feel we can enumerate schema types from task context here since data processor can only handle logic for input=>output?

Reply:
Sure, have updated.